modules/govuk/manifests/node/s_whitehall_mysql_master.pp

```diff
@@ -36,4 +52,13 @@
   }
 
   Govuk_mount['/var/lib/mysql'] -> Class['govuk_mysql::server']
+
+  if $s3_bucket_name {
```

Comment:
@rjw1 mentioned that it might be good to have _all_ the required pieces defined in the conditional here: the define will already [get upset if we don't already have all the parts](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_mysql/manifests/xtrabackup/backup.pp#L37-L40), so the conditional is there just to ensure Puppet doesn't break if we have a bucket defined. I just wonder if it's worth having a long  line of `if $s3_bucket_name and $aws_access_key_id and $aws_secret_access_key and $encryption_key {` for this purpose?